manager/common/license.go

```diff
@@ -1,40 +1,41 @@
 package license
+
 import (
+	astoreServer "github.com/enfabrica/enkit/astore/server/astore"
+	rpc_license "github.com/enfabrica/enkit/manager/rpc"
+	"google.golang.org/grpc/metadata"
 	"io"
 	"log"
+	"math/rand"
+	"strings"
 	"sync"
 	"time"
-	"strings"
-	"math/rand"
-	"google.golang.org/grpc/metadata"
-	rpc_license "github.com/enfabrica/enkit/manager/rpc"
-	astoreServer "github.com/enfabrica/enkit/astore/server/astore"
 )
 
 type LicenseCounter struct {
-    counterMutex sync.Mutex
-    counter map[string]int // current number of licenses used per vendor
-	clientMutex sync.Mutex
-	clients map[string]*Client // client connection metadata
-    total map[string]int // total licenses available per vendor
-	queue []string
+	counterMutex sync.Mutex
+	counter      map[string]int // current number of licenses used per vendor
```

Comment:
I recommend having counter and total part of the same map. A single map cannot go out of sync :)
eg
```
license map[string]*License
```

```
type License struct {
  Total int
  Used int
}
```